base_tier_validation/templates/tier_validation_templates.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="tier_validation_buttons">
+        <div>
+            <button
+                name="request_validation"
+                string="Request Validation"
+                t-attf-invisible="not need_validation or rejected or hide_reviews"
+                type="object"
+            />
+            <button
+                name="restart_validation"
+                string="Restart Validation"
+                invisible="not review_ids or hide_reviews"
+                type="object"
+            />
+        </div>
+    </template>
+    <template id="tier_validation_label">
+        <div>
+            <field name="hide_reviews" invisible="1" />
+            <field name="need_validation" invisible="1" />
+            <field name="validated" invisible="1" />
+            <field name="rejected" invisible="1" />
+            <div
+                class="alert alert-warning"
+                role="alert"
+                t-attf-invisible="validated or hide_reviews or rejected or not review_ids"
+                style="margin-bottom:0px;"
```

Comment:
```suggestion
                style="margin-bottom:0;"
```
or just simple to remove style attrs and add mb-0 include class attrs params like: `class="alert alert-warning mb-0"`